include/devicetree.h

```diff
@@ -2654,6 +2671,16 @@
  */
 #define DT_INST_ON_BUS(inst, bus) DT_ON_BUS(DT_DRV_INST(inst), bus)
 
+/**
+ * @brief Does a DT_DRV_COMPAT instance have string token?
```

Comment:
This brief is strange, it sounds like the macro returns `true`/`false`.

Reply:
Done/